### DISCUSSION TITLE
How can I achieve fusion of two consecutive GEMMs using CUTE?

### DISCUSSION THREAD

**Discussion by luliyucoordinate** [NVIDIA/cutlass#1720] [Q&A] [answered]
I am trying to implement fusion of two consecutive GEMMs (mxn * nxk * kxn) using cute. How can I set the result of the first GEMM with tiled MMA to be compatible with the second GEMM? In other words, how can I achieve something similar to examples/13_two_tensor_op_fusion using cute?

**Comment by thakkarV** [comment, marked as answer]
https://github.com/Dao-AILab/flash-attention/tree/main/hopper

this is the best example we have
